### DISCUSSION TITLE
Demo Video Dotfiles

### DISCUSSION THREAD

**Discussion by unrealapex** [pystardust/ani-cli#1237] [Q&A] [answered]
The rice displayed on the demo video looks very nice. Is there a chance that someone has the dotfiles available?

https://user-images.githubusercontent.com/44473782/224679247-0856e652-f187-4865-bbcf-5a8e5cf830da.webm

**Comment by port19x** [comment]
Yes, those are the dots of @justchokingaround
[his dots](https://github.com/justchokingaround/dotfiles)

**Reply by unrealapex** [reply]
Thank you for the response! Looking through the commits, unfortunately, I didn't find what I was looking for. Specifically, I was wondering what border color he was using in the rice. I'm using the same wallpaper and wanted to take some inspiration. Attempting to extract it, the color I got was `#bf4e6d`, but I think this is inaccurate given the video has some compression.

**Reply by CoolnsX** [reply]
```
rgba(D81860FF)
```

**Reply by unrealapex** [reply]
The hex code your provided doesn't appear to be the same color.

**Reply by justchokingaround** [reply, marked as answer]
its `#ee5396` just ping me if u need anything, i dont usually look through threads @unrealapex

**Reply by unrealapex** [reply]
Thank you for the response! I don't mean to be picky, but are you certain that is the color? It seems to be a different hue from that shown in the video @justchokingaround.

**Reply by CoolnsX** [reply]
Bro he made the video

**Reply by unrealapex** [reply]
I am well aware that he made the video. My apologies for being a perfectionist.
